README.md

```diff
@@ -1 +1,170 @@
-# inheritance
\ No newline at end of file
+# Grocery Store
+
+Let's simulate a grocery store system! We want to be able to keep track of the orders that folks make, both online and physically in our grocery store.
+
+This project will allow you to explore object-oriented design as well as a few other new topics.
+
+## Baseline Setup
+
+1. This is an individual, stage 1 project.
+1. Fork the project master.
+1. Clone the forked repo: `$ git clone [YOUR FORKED REPO URL]`
+1. `cd` into the dir created `$ cd grocery-store`
+1. Run `git remote -v` to verify the folder you are in corresponds to the fork you have created.
+1. Run `gem install minitest-skip` to install an extra gem for testing (more on what this actually does later).
+
+### Testing
+
+This is our first project with real tests! Following the instructions from the [TDD lecture](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/intro-to-automated-tests.md), there are three things in our project directory:
+
+```
+Rakefile
+lib/
+specs/
+```
+
+Each class you write (there will only be one until wave 3) should get its own file, `lib/class_name.rb`. The specs for that class will be in `specs/class_name_spec.rb`, and you can run all specs using `rake`.
+
+For wave 1, all tests will be given to you - your job is to write code to make them pass. For waves 2 and 3, we supply descriptions of the tests, but you have to write them yourself.
+
+## Wave 1
+
+### Learning Goals
+- Create a **class** inside of a **module**
```

Comment:
I seem to remember we kinda left out modules last cohort.  Are we introducing them for C8?  I think we should, just checking.